src/components/Button.js

```diff
@@ -183,7 +179,6 @@ class Button extends Component {
 
         const textComponent = (
             <Text
-                selectable={this.props.textSelectable}
```

Comment:
It looks like this was `selectable={false}` [before the PR](https://github.com/Expensify/App/pull/11367/files#diff-790c64a0c38ecc16de3bba948ef8d164ac9678900a661b39ae97715bcf4ca6deL182), rather than not there at all - do we need to update?

Reply:
Oh, good spot. Thank you

Reply:
Updated 👍